docs/source/user_guide.rst

```diff
@@ -167,6 +167,38 @@ we can now retrieve the position of the ISS after some time::
 For more advanced propagation options, check out the
 :py:mod:`poliastro.twobody.propagation` module.
 
+Accounting non-keplerian orbits: perturbation
+---------------------------------------------
+
+poliastro has recently got the module accounting perturbations
+to the Newton gravitational law. The perturbation acceleration
+should be passed to the ``cowell`` propagation method along
+with the parameters describing the disturbing force. For instance,
+let us examine the effect of J2 perturbation::
+
+    >>> from poliastro.twobody.perturbations import J2_perturbation
+    >>> from poliastro.twobody.propagation import cowell
+    >>> from poliastro.twobody.rv import rv2coe
+    >>> r0 = np.array([-2384.46, 5729.01, 3050.46])  # km
+    >>> v0 = np.array([-7.36138, -2.98997, 1.64354])  # km/s
+    >>> orbit = Orbit.from_vectors(Earth, r0 * u.km, v0 * u.km / u.s)
+    >>> tof = (48.0 * u.h).to(u.s)
+    >>> orbit = orbit.propagate(tof, method=cowell, ad=J2_perturbation, J2=Earth.J2.value, R=Earth.R.to(u.km).value)
+    >>> r, v = orbit.rv()
+
+The J2 perturbation changes the orbit parameters (from Curtis example 12.2)::
+
+    >>> k = Earth.k.to(u.km**3 / u.s**2).value
+    >>> _, _, _, raan0, argp0, _ = rv2coe(k, r0, v0)
```

Comment:
Sorry for being a pain in the neck, but if we have both initial and final `Orbit` objects, we do not need the `rv2coe` either. You can just do

```
final.raan - initial.raan
```